shell-completion/bash/systemd-analyze

```diff
@@ -225,6 +226,13 @@ _systemd_analyze() {
         if [[ $cur = -* ]]; then
             comps='--help --version --no-pager --json=off --json=pretty --json=short -m --mask'
         fi
+
+    elif __contains_word "$verb" ${VERBS[TRANSIENT_SETTINGS]}; then
+        if [[ $cur = -* ]]; then
+            comps='--help --version --no-pager --json=off --json=pretty --json=short -m --mask'
```

Comment:
`--json=` and `--mask/-m` are not supported, I guess. Please drop them.

Also, `--no-pager` support is missing, and I think it should be added.